src/Components/Results.js

```diff
@@ -0,0 +1,21 @@
+// Results based on user's input 
+const Results = (props) => {
+
+   return (
+      <section className="wrapper" key={props.id}>
+         <h3>{props.name}</h3>
+
+         <div className="fullInfo">
+            <img src={props.images} alt=" poster for the event" />
```

Comment:
for accessibility purposes, maybe the alt text on the image should be more specific such as "poster for <event name>"

Reply:
"poster for _event name_"